FinSearchUnified/Subscriber/Frontend.php

```diff
@@ -219,4 +233,28 @@ private function isManufacturerPage(Request $request)
     {
         return $request->getControllerName() === 'listing' && $request->getActionName() === 'manufacturer';
     }
+
+    /**
+     * @param Request $request
```

Comment:
The docblock and the type hint do not match. The actual type is `Enlight_Controller_Request_Request` and not `Request`.